### ISSUE TITLE
Error "kubectl is outdated"

### ISSUE THREAD

**Issue by joalder** [adrianliechti/devkube#27]
Tried creating a cluster using devkube in WSL supported by Docker Desktop and just got the following.

```
# devkube create playground
✔ local
kubectl is outdated. see https://kubernetes.io/docs/tasks/tools/install-kubectl
```

```
# kubectl version
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.2", GitCommit:"5835544ca568b757a8ecae5c153f317e5736700e", GitTreeState:"clean", BuildDate:"2022-09-21T14:25:45Z", GoVersion:"go1.19.1", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v4.5.7
```

```
#  devkube --version
devkube version 0.1.1
```

```
# uname -a
Linux PC-123 5.10.102.1-microsoft-standard-WSL2 #1 SMP Wed Mar 2 00:30:59 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

Creation worked just fine manually using kind `0.16.0`.

```
# kind create cluster -n test
Creating cluster "test" ...
 ✓ Ensuring node image (kindest/node:v1.25.2) 🖼
 ✓ Preparing nodes 📦
 ✓ Writing configuration 📜
 ✓ Starting control-plane 🕹️
 ✓ Installing CNI 🔌
 ✓ Installing StorageClass 💾
Set kubectl context to "kind-test"
You can now use your cluster with:

kubectl cluster-info --context kind-test

Not sure what to do next? 😅  Check out https://kind.sigs.k8s.io/docs/user/quick-start/
```

**Comment by adrianliechti** [comment]
let's have a debug session then :)

but since I am curious already:

* how have you installed devkube?
* how have you installed kubectl?
* how have you installed helm?
* which wsl distribution do you use?

can you send me the output of:

```bash
$ which devkube
/opt/homebrew/bin/devkube

$ which kubectl
/opt/homebrew/bin/kubectl

$ which helm
/opt/homebrew/bin/kubectl

$ kubectl version --client -o json
{
  "clientVersion": {
    "major": "1",
    "minor": "25",
    "gitVersion": "v1.25.2",
    "gitCommit": "5835544ca568b757a8ecae5c153f317e5736700e",
    "gitTreeState": "clean",
    "buildDate": "2022-09-21T14:25:45Z",
    "goVersion": "go1.19.1",
    "compiler": "gc",
    "platform": "darwin/arm64"
  },
  "kustomizeVersion": "v4.5.7"
}
```

**Comment by adrianliechti** [comment]
created a fresh WSL / Debian instance, and do see the same weird error...

as a workaround, try the following:

```bash
# this will fail - but creates the cluster (currently the same as `kind create cluster --name devkube`)
$ devkube create

# export the kubeconfig file of the failed but running cluster (currently same as `kind export kubeconfig --name devkube`)
$ devkube setup

# this will use the current kubeconfig - and install everything else on it (also allowing you to BYOD cluster for tests)
$ devkube create --provider none
```

I will try to reproduce the issue. Does it work when using the Windows Version (e.g. via scoop) for you?

**Comment by adrianliechti** [comment]
we get an error of `context canceled` while continue work...
so basically, the cli framework thinks the user or system pressed ctrl-c or similar

**Comment by adrianliechti** [comment]
Hopefully fixed in #28 / Version v0.1.2. Let me know!
Cheers


PS: I actually blame copilot for that - and my super fast tab-press finger

**Comment by joalder** [comment]
Works great, thanks a lot.
